api/v1alpha1/networkchaos_webhook.go

```diff
@@ -44,6 +52,20 @@ func (in *NetworkChaos) Default() {
 	in.Spec.Selector.DefaultNamespace(in.GetNamespace())
 	// the target's namespace selector
 	in.Spec.Target.TargetSelector.DefaultNamespace(in.GetNamespace())
+
+	in.defaultDelay()
```

Comment:
Shall we add it to `NetworkChaosSpec` to keep same style as other's setting default ?

Reply:
I agree with this opinion @Gallardot

Reply:
Good suggestion!All done!